scripts/data/cleanWDPA.R

```diff
@@ -0,0 +1,195 @@
+library(sf)
+library(rjson)
+library(dplyr)
+library(wdpar)
+
+# Add inputs
+input <- biab_inputs()
+sf_use_s2(FALSE)
+# Get polygon for study area
+
+# Read in polygon of study area
+if(is.null(input$study_area_polygon)){
+  biab_error_stop("No study area polygon found. Please provide a geopackage of the study area. To pull country or region 
+  shapefiles, connect this script to the 'Get country polygon' script")
+  }
+
+study_area <- input$study_area_polygon
+study_area <- sf::st_read(study_area, type=3, promote_to_multi=FALSE) 
+study_area <- st_transform(study_area, crs=input$crs)
+stidy_area <- st_make_valid(study_area)
```

Comment:
Should be "study_area"
```suggestion
study_area <- st_make_valid(study_area)
```